dblookupext/esdtSupply/esdtSuppliesProcessor_test.go

```diff
@@ -0,0 +1 @@
+package esdtSupply
```

Comment:
missing tests. This is targeted towards development branch so it should contain tests.

Reply:
added